### PR TITLE
sqsc: add page

### PR DESCRIPTION
**Pull request by mattbangert**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/sqsc.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS SQS client.
```
Comment (sbrl) [inline pages/common/sqsc.md]: What does `SQS` stand for?

File: pages/common/sqsc.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Copy all messages from one queue to another:
+
+`sqsc cp <from-queue-name> <to-queue-name>`
+
+- Move all messages from one queue to another:
+
+`sqsc mv <from-queue-name> <to-queue-name>`
+
+- Descibe a queue:
```
Comment (sbrl) [inline pages/common/sqsc.md]: Typo - `Descibe` -> `Describe`?

Also, what do you mean by `describe`?

File: pages/common/sqsc.md (hunk 1 above)
Comment (mattbangert) [reply pages/common/sqsc.md]: Simple Queue Service, basically hosted ActiveMQ.

File: pages/common/sqsc.md (hunk 2 above)
Comment (mattbangert) [reply pages/common/sqsc.md]: Updated.  Describes a queue's attributes

image

File: pages/common/sqsc.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- Describe a queue:
+
+`sqsc describe <queue-name>`
+
+- Query a queue with SQL syntax:
+
+`sqsc query "SELECT body FROM <queue-name> WHERE body LIKE '%user%'"`
+
+- Pull all messages in a queue into a local sqlite database:
```
Comment (agnivade) [inline pages/common/sqsc.md]: "from a queue".

Also, where is the sqlite db file specified ?

File: pages/common/sqsc.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+`sqsc lq [queue-prefix]`
+
+- List all messages in a queue:
+
+`sqsc ls <queue-name>`
+
+- Copy all messages from one queue to another:
+
+`sqsc cp <from-queue-name> <to-queue-name>`
```
Comment (agnivade) [inline pages/common/sqsc.md]: source_queue, destination_queue

File: pages/common/sqsc.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- Move all messages from one queue to another:
+
+`sqsc mv <from-queue-name> <to-queue-name>`
```
Comment (agnivade) [inline pages/common/sqsc.md]: same as above.

File: pages/common/sqsc.md (hunk 1 above)
Comment (sbrl) [reply pages/common/sqsc.md]: Ah, ok! Perhaps we could update the description then?

File: pages/common/sqsc.md (hunk 4 above)
Comment (mattbangert) [reply pages/common/sqsc.md]: Updated

File: pages/common/sqsc.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS Simple Queue Service client.
+
+- List all queues:
+
+`sqsc lq [queue-prefix]`
```
Comment (sbrl) [inline pages/common/sqsc.md]: I think @agnivade was trying to also ask you to change out the `[` and `]` for `{{` and `}}`.

File: pages/common/sqsc.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+- List all messages in a queue:
+
+`sqsc ls <queue-name>`
```
Comment (sbrl) [inline pages/common/sqsc.md]: Same here. `<queue-name>` -> `{{queue-name}}`. The same also goes for the other examples further down.

**Pull request merged**